devops/docker/tasking-manager/Dockerfile

```diff
@@ -1,31 +1,44 @@
-# Base on Python on Debian stable
 FROM python:3-stretch
+MAINTAINER Tasking Manager development team
 
-EXPOSE 5000
+## API/SERVER CONFIGURATION
 
-# Install dependencies
+# Install dependencies for shapely
 RUN apt-get update \
   && apt-get upgrade -y \
   && apt-get install -y libgeos-dev \
   && apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 
-RUN curl -sL https://deb.nodesource.com/setup_9.x | bash - \ 
+RUN mkdir -p /usr/src/app
+WORKDIR /usr/src/app
+
+# Upgrade pip
+RUN pip install --upgrade pip
+
+ARG branch=develop
```

Comment:
Same here and next two lines. I think this is more general and doesn't fit under `## SERVER CONFIGURATION`

Reply:
Check